### PR TITLE
Resurrect changes from #8131

### PR DESCRIPTION
**Pull request by markstory**
These changes were lost when we moved to the 3.next branch. This revives those changes and solves #8259

### CODE REVIEW

File: src/Error/ExceptionRenderer.php [hunk 1]
```diff
@@ -183,7 +183,6 @@ public function render()
                 'format' => 'array',
                 'args' => false
             ]);
-            $viewVars['_serialize'][] = 'trace';
```
Comment (ADmad) [inline src/Error/ExceptionRenderer.php]: I like having traces in json output :(

**Comment by dereuromark** [comment]
Looks good.

File: src/Error/ExceptionRenderer.php (hunk 1 above)
Comment (markstory) [inline src/Error/ExceptionRenderer.php]: As before though, they sometimes cause fatal errors. The stack trace is still available in the logs.

Comment (jhli) [inline src/Error/ExceptionRenderer.php]: I like them too, however xml bugged out.
Should I add the info to the book @ logs in this case?

Comment (markstory) [inline src/Error/ExceptionRenderer.php]: Sure. Educating people on log files is probably a good idea.

**Pull request merged**